application/src/main/java/org/thingsboard/server/controller/SystemInfoController.java

```diff
@@ -80,6 +80,12 @@ public class SystemInfoController extends BaseController {
     @Value("${actors.rule.chain.debug_mode_rate_limits_per_tenant.configuration:50000:3600}")
     private String ruleChainDebugPerTenantLimitsConfiguration;
 
+    @Value("${actors.calculated_fields.debug_mode_rate_limits_per_tenant.enabled:true}")
+    private boolean calculatedFieldDebugPerTenantLimitsEnabled;
```

Comment:
Please extract 

```
    @Value("${actors.rule.chain.debug_mode_rate_limits_per_tenant.enabled:true}")
    private boolean ruleChainDebugPerTenantLimitsEnabled;
    @Value("${actors.rule.chain.debug_mode_rate_limits_per_tenant.configuration:50000:3600}")
    private String ruleChainDebugPerTenantLimitsConfiguration;

    @Value("${actors.calculated_fields.debug_mode_rate_limits_per_tenant.enabled:true}")
    private boolean calculatedFieldDebugPerTenantLimitsEnabled;

    @Value("${actors.calculated_fields.debug_mode_rate_limits_per_tenant.configuration:50000:3600}")
    private String calculatedFieldDebugPerTenantLimitsConfiguration;
```

into a component and use it here and in ActorSystemContext